src/lib/PreviewError.js

```diff
@@ -17,12 +17,13 @@ class PreviewError extends Error {
      * @param {string} code - Error code
      * @param {string} [displayMessage] - Translated error message that can be displayed to a user
      * @param {Object} [details] - Optional error options
-     * @param {...Object} [params] - Additional params, this may contain the original error message
+     * @param {string} [message] - Original error message
+     * @param {...Object} [params] - Additional params
      * @return {PreviewError} PreviewError instance
      */
-    constructor(code, displayMessage = '', details = {}, ...params) {
+    constructor(code, displayMessage = '', details = {}, message, ...params) {
         // Pass remaining arguments to parent constructor
-        super(...params);
+        super(message, ...params);
```

Comment:
Since the parent constructor wants [message, fileName, lineNumber], you can probably get rid of ...params since latter two are optional and not passed in for PreviewErrors usage.

Reply:
Yeah, I didn't see them passed in anywhere, but I wanted to stay as close to the original implementation as possible.

Reply:
Also PreviewError's 2nd arg is the message, aka `displayMessage`, so shouldn't it be super(displayMessage)?

Reply:
Nope. If both are passed in, `this.message` should still be the original message, not the display message.